libraries/cmake/source/augeas/README.md

```diff
@@ -61,55 +62,49 @@ Copy the generated gnulib files:
  - `config.h` (use the augeas one)
  - All the files containing `AUTOGENERATED` under `lib`
 
-# macOS
```

Comment:
nit: Why two # here (and the additional # below)? This not a sub-heading of Linux